app/src/main/java/com/github/se/eduverse/ui/todo/TodoListScreen.kt

```diff
@@ -112,11 +152,8 @@ fun TodoItem(todo: Todo, onUndo: (Todo) -> Unit, onDone: (Todo) -> Unit) {
                       .testTag("todoName_${todo.uid}"),
               textDecoration = if (completed) TextDecoration.LineThrough else TextDecoration.None,
               color = if (completed) Color.Gray else Color.Unspecified)
-          IconButton(
-              onClick = {},
-              modifier = Modifier.testTag("todoOptionsButton_${todo.uid}").padding(8.dp)) {
-                Icon(Icons.Default.MoreVert, contentDescription = "Todo Options")
-              }
+
+          rightMostButton()
         }
```

Comment:
A lot of values are hard-coded, be careful that it works also on phone with a different size than yours